lib/irb/input-method.rb

```diff
@@ -470,4 +470,13 @@ def inspect
       str
     end
   end
+
+  class ReidlineInputMethod < RelineInputMethod
```

Comment:
Sorry that I missed this definition. I think it's a safe change and added #411  to pick up a few leftovers.